tests/test_cli.py

```diff
@@ -0,0 +1,60 @@
+
+from  __future__ import unicode_literals
+import time
+
+import jwt
+from jwt.__main__ import build_argparser, decode_payload, encode_payload, main
+import pytest
+
+
+class TestCli:
+
+    def test_build_argparse(self):
+        args = ['--key', '1234', 'encode', 'name=Vader']
+        parser = build_argparser()
+        parsed_args = parser.parse_args(args)
+
+        assert parsed_args.key == '1234'
+
+    def test_encode_payload_raises_value_error_key_is_required(self):
+        encode_args = ['encode', 'name=Vader', 'job=Sith']
+        parser = build_argparser()
+
+        args = parser.parse_args(encode_args)
+
+        with pytest.raises(ValueError) as excinfo:
+            encode_payload(args)
+
+        assert 'Key is required when encoding' in str(excinfo.value)
+
+    def test_decode_payload_raises_decoded_error(self):
+        decode_args = ['--key', '1234', 'decode', 'wrong-token']
+        parser = build_argparser()
+
+        args = parser.parse_args(decode_args)
+
+        with pytest.raises(jwt.DecodeError) as excinfo:
+            decode_payload(args)
+
+        assert 'There was an error decoding the token' in str(excinfo.value)
+
+    @pytest.mark.parametrize('key,name,job', [
+        ('1234', 'Vader', 'Sith'),
+        ('4567', 'Anakin', 'Jedi'),
+    ])
+    def test_main_run(self, key, name, job):
+        args = [
```

Comment:
Two errors here in 2.6 https://travis-ci.org/jpadilla/pyjwt/jobs/232649652

Changing it to the following should fix.

```python
args = [
    '--key', key,
    'encode',
    'name={0}'.format(name),
    'job={0}'.format(job)
]
```